setup.py

```diff
@@ -10,7 +10,7 @@
 
 setup(
     name="pyinstrument",
-    packages=find_packages(),
+    packages=find_packages(exclude=("test","test.*")),
```

Comment:
Wouldn't it be better to only `include='pyinstrument'` instead? This would still include any other directory that happens to contain an `__init__.py`.

Reply:
Sure, that’s reasonable. I’ll force-push an updated version that does that.